package.json

```diff
@@ -134,9 +132,8 @@
     "coveralls": "^3.0.1",
     "eslint": "^5.7.0",
     "eslint-plugin-prettier": "^3.0.0",
-    "firebase-admin": "^7.3.0",
+    "firebase-admin": "^7.0.0",
```

Comment:
any objection to bringing this up to the latest version of `firebase-admin` (`^8.0.0`)?

Reply:
@abeisgoat ?

Reply:
+1 on this, functions also changing to 8.0.0, if we can here too that would be great. I believe the biggest change there is dropping Node 6 support.

Reply:
Abe is on vacation but I think this would be a good idea.

Reply:
Easy breezy: https://github.com/firebase/firebase-tools/pull/1393